src/server/home/controller.js

```diff
@@ -6,7 +8,7 @@
 export const homeController = {
   handler(_request, h) {
     return h.view('home/index', {
-      pageTitle: 'Apply for a Bovine Tuberculosis (TB) movement licence',
+      pageTitle: config.get('serviceName'),
```

Comment:
Is it going to change between envs? If not we could just set the actual value of the title here.

Reply:
This is one of those cases where we want something to be invarient - no matter the environment. 

If we have it in config, it weakens our tests (e.g. sure it's checked the service in the publish checker - but what test says that's the service name in production?).

IMO, this is definitely best to left hardcoded - either directly, or in a `const` somewhere in the code.

Reply:
to be clear you are asking be to go through everywhere the title config prop is used and hard code it? totally fine, just making sure i understand the ask